### PR TITLE
created my username to be added to eddiehub

### PR DESCRIPTION
**Pull request by ShlokMohanty**
in this i have added my username and social media handles for being added to the eddiehub as a new and as a beginner contributor to eddiehubs repo.

Fixes #

## Proposed Changes

  -
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/223"></a>

### CODE REVIEW

**Comment by ShlokMohanty** [comment]
Should I change my bio to something else instead of Member of Eddiehub?

**Review by FayasNoushad — COMMENTED** [review]
Facebook icon and url is empty

**Comment by ShlokMohanty** [comment]
@FayasNoushad now can i see my changes on the facebook URL and icon?

**Comment by CodingSpecies** [comment]
LGTM looks good now.

File: public/data/ShlokMohanty.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "Follow me on github",
+      "url": "http://github.com/ShlokMohanty",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mohanty_shlok",
+      "icon": "twitter"
+    }
```
Comment (rohan-kulkarni-25) [inline public/data/ShlokMohanty.json:15]: Please add comma after object

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]

**Comment by ShlokMohanty** [comment]
@rohan-kulkarni-25 is now everything fine added the comma on line number 15

**Review by rohan-kulkarni-25 — APPROVED** [review]
LGTM 💥

**Review by FayasNoushad — APPROVED** [review]

**Comment by ShlokMohanty** [comment]
@rohan-kulkarni-25 thanks for approving

**Comment by ShlokMohanty** [comment]
@FayasNoushad thanks for your approval

**Review by aayush89890 — APPROVED** [review]
Welcome to the community.

**Comment by ShlokMohanty** [comment]
@aayush89890  thanks for your warm welcome

File: public/data/ShlokMohanty.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "icon": "twitter"
+    },
+   {
+    "name": "Follow me on Facebook",
+    "url": "https://www.facebook.com/shlok.mohanty.35",
+    "icon": "facebook"
+    }
+  ]
+}
+
```
Comment (Panquesito7) [inline public/data/ShlokMohanty.json:23]:
```suggestion
```

File: public/data/ShlokMohanty.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub",
+  "avatar": "https://github.com/ShlokMohanty.png",
+  "links": [
+    {
+      "name": "Follow me on github",
+      "url": "http://github.com/ShlokMohanty",
```
Comment (Panquesito7) [inline public/data/ShlokMohanty.json:8]:
```suggestion
      "url": "https://github.com/ShlokMohanty",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/ShlokMohanty.json (hunk 2 above)
Comment (ShlokMohanty) [reply public/data/ShlokMohanty.json:23]: @Panquesito7 do i have to make any changes or changes have already been made?

File: public/data/ShlokMohanty.json (hunk 3 above)
Comment (ShlokMohanty) [reply public/data/ShlokMohanty.json:8]: @Panquesito7 i guess you made the change from https to http or it still needs to be done ?

Comment (FayasNoushad) [reply public/data/ShlokMohanty.json:8]: No @ShlokMohanty, He just suggested.

Comment (FayasNoushad) [reply public/data/ShlokMohanty.json:8]: I also recommending that.

File: public/data/ShlokMohanty.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      "name": "Follow me on github",
+      "url": "https://github.com/ShlokMohanty",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/mohanty_shlok",
+      "icon": "twitter"
+    },
+   {
```
Comment (FayasNoushad) [inline public/data/ShlokMohanty.json:16]: Line 16 add one white space too before `{`

File: public/data/ShlokMohanty.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "Follow me on Facebook",
+    "url": "https://www.facebook.com/shlok.mohanty.35",
+    "icon": "facebook"
```
Comment (FayasNoushad) [inline public/data/ShlokMohanty.json:19]: Add 2  white space before these dict variables

Comment (FayasNoushad) [reply public/data/ShlokMohanty.json:19]: Line number 17, 18, and 19

Comment (ShlokMohanty) [reply public/data/ShlokMohanty.json:19]: @FayasNoushad made the changes can you see it in the pr generated ?

File: public/data/ShlokMohanty.json (hunk 4 above)
Comment (ShlokMohanty) [reply public/data/ShlokMohanty.json]: i think i have made the changes @FayasNoushad?

File: public/data/ShlokMohanty.json (hunk 2 above)
Comment (FayasNoushad) [reply public/data/ShlokMohanty.json:23]: Resolve this conversation

File: public/data/ShlokMohanty.json (hunk 4 above)
Comment (FayasNoushad) [reply public/data/ShlokMohanty.json]: Ok

**Comment by ShlokMohanty** [comment]
@FayasNoushad should i resolve the conversation?

File: public/data/ShlokMohanty.json (hunk 3 above)
Comment (ShlokMohanty) [reply public/data/ShlokMohanty.json]: @FayasNoushad  should i also resolve this conversation as well ?

**Comment by krishdevdb** [comment]
Yeah, Mark it as resolved, that would be really helpful 👍🏻

**Comment by ShlokMohanty** [comment]
@krishdevdb thanks resolved the conversation

**Review by krishdevdb — APPROVED** [review]
LGTM

File: public/data/ShlokMohanty.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub",
```
Comment (Panquesito7) [inline public/data/ShlokMohanty.json:3]: Maybe describe a bit of yourself here instead of just `Member of EddieHub`?

Comment (eddiejaoude) [reply public/data/ShlokMohanty.json:3]: This would be useful 👍 , but can come in another PR

**Comment by eddiejaoude** [comment]
I will close as looks like this already exists https://github.com/EddieHubCommunity/LinkFree/blob/main/public/data/ShlokMohanty.json

**Pull request closed without merging**